### PR TITLE
chore(app): update backend README.md, setup docker for running backend

### PR DESCRIPTION
**Pull request by vinujosep**

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,17 @@
 FROM node:10
+
```
Comment (ohel) [inline backend/Dockerfile]: Why are you adding empty lines everywhere? The point of empty lines is to group code and commands to be more readable. Adding them everywhere just makes things harder to read.

File: backend/README.md [hunk 2]
```diff
@@ -1,79 +1,65 @@
-To generate documenation used `npm run compodoc:build` to serve documentation use `npm run compodoc`
+`/backend` includes the server side code for running Mobiilinutakortti app.
```
Comment (ohel) [inline backend/README.md]: Nitpicking, but when referring to directories, starting with slash might be triggering. In this case it's a relative path, so ./backend is more suitable, or simply "The `backend` directory --".

File: backend/Dockerfile (hunk 1 above)
Comment (vinujosep) [reply backend/Dockerfile]: The idea was to make it more readable in code, but i understand it's difficult while reviewing. Will fix it and update right away

File: backend/README.md [hunk 3]
```diff
@@ -1,79 +1,65 @@
...
-<a href="https://gitter.im/nestjs/nestjs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge"><img src="https://badges.gitter.im/nestjs/nestjs.svg" alt="Gitter" /></a>
-<a href="https://opencollective.com/nest#backer"><img src="https://opencollective.com/nest/backers/badge.svg" alt="Backers on Open Collective" /></a>
-<a href="https://opencollective.com/nest#sponsor"><img src="https://opencollective.com/nest/sponsors/badge.svg" alt="Sponsors on Open Collective" /></a>
-  <a href="https://paypal.me/kamilmysliwiec"><img src="https://img.shields.io/badge/Donate-PayPal-dc3d53.svg"/></a>
-  <a href="https://twitter.com/nestframework"><img src="https://img.shields.io/twitter/follow/nestframework.svg?style=social&label=Follow"></a>
-</p>
-  <!--[![Backers on Open Collective](https://opencollective.com/nest/backers/badge.svg)](https://opencollective.com/nest#backer)
-  [![Sponsors on Open Collective](https://opencollective.com/nest/sponsors/badge.svg)](https://opencollective.com/nest#sponsor)-->
+<br/>
```
Comment (ohel) [inline backend/README.md]: These shouldn't be used in a Markdown document I think :o

**Pull request merged**